### DISCUSSION TITLE
SFSC - SFCC Connector from storefront

### DISCUSSION THREAD

**Discussion by shayunBBD** [SalesforceCommerceCloud/sfcc-ci#429] [Team Posts]
Hello all,

I succeed to install the connector but my sync status is always set to None.
If I changed it to created and execute the sync job, it works and the new records are created in service cloud.
I understand that if I try to create a new customer from my storefront it should be created with sync-status = created. But it doesn't and also is not sent to SC.
I saw in the logs and found this error, can you please help me?

ERROR PipelineCallServlet|1902034155|Sites-...-Site|Account-StartRegister|PipelineCall|IGnJwBaYg0 custom.CRM []  SoapRequest.CreateRequest - Request :

The request is a xml with my customer details

`<?xml version="1.0" encoding="UTF-8"?>
<s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" xmlns:a="http://www.w3.org/2005/08/addressing">
   <s:Body>
      <Execute xmlns="http://schemas.microsoft.com/xrm/2011/Contracts/Services">
         <request xmlns:b="http://schemas.microsoft.com/xrm/2011/Contracts" xmlns:i="http://www.w3.org/2001/XMLSchema-instance" i:type="b:CreateRequest">
            <b:Parameters xmlns:c="http://schemas.datacontract.org/2004/07/System.Collections.Generic" xmlns:d="http://www.w3.org/2001/XMLSchema" xmlns:g="http://schemas.microsoft.com/2003/10/Serialization/">
               <b:KeyValuePairOfstringanyType>
                  <c:key>Target</c:key>
                  <c:value i:type="b:Entity">
                     <b:Attributes>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_contactid</c:key>
                           <c:value i:type="d:string"><![CDATA[My cust no]]></c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>lastname</c:key>
                           <c:value i:type="d:string" />
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>firstname</c:key>
                           <c:value i:type="d:string" />
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>emailaddress1</c:key>
                           <c:value i:type="d:string"><![CDATA[My email]]></c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>telephone1</c:key>
                           <c:value i:type="d:string" />
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>mobilephone</c:key>
                           <c:value i:type="d:string" />
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>ownerid</c:key>
                           <c:value i:type="b:EntityReference">
                              <b:Id>Id</b:Id>
                              <b:LogicalName>systemuser</b:LogicalName>
                              <b:Name>CRM System</b:Name>
                           </c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_navigationlanguage</c:key>
                           <c:value i:type="b:OptionSetValue">
                              <b:Value>0</b:Value>
                           </c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_language</c:key>
                           <c:value i:type="b:OptionSetValue">
                              <b:Value>0</b:Value>
                           </c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_siteorstore</c:key>
                           <c:value i:type="b:EntityReference">
                              <b:Id>Id</b:Id>
                              <b:LogicalName>businessunit</b:LogicalName>
                              <b:Name>Individual My Site</b:Name>
                           </c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_siteorstorecurrent</c:key>
                           <c:value i:type="b:EntityReference">
                              <b:Id>My Id</b:Id>
                              <b:LogicalName>businessunit</b:LogicalName>
                              <b:Name>Individual My Site</b:Name>
                           </c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_newsletter</c:key>
                           <c:value i:type="d:boolean">false</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_storenewsletter</c:key>
                           <c:value i:type="d:boolean">false</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_optinpartner</c:key>
                           <c:value i:type="d:boolean">true</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_sms</c:key>
                           <c:value i:type="d:boolean">true</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_dwcreationdate</c:key>
                           <c:value i:type="d:dateTime">2019-07-03T15:08:56</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_lastlogindate</c:key>
                           <c:value i:type="d:dateTime">2019-07-03T15:09:03</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_olderthan7days</c:key>
                           <c:value i:type="d:boolean">true</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_isweb</c:key>
                           <c:value i:type="d:boolean">true</c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_sourcechannel</c:key>
                           <c:value i:type="b:OptionSetValue">
                              <b:Value>1</b:Value>
                           </c:value>
                        </b:KeyValuePairOfstringanyType>
                        <b:KeyValuePairOfstringanyType>
                           <c:key>it_webjoinon</c:key>
                           <c:value i:type="d:dateTime">2019-07-03T15:09:03</c:value>
                        </b:KeyValuePairOfstringanyType>
                     </b:Attributes>
                     <b:EntityState>Created</b:EntityState>
                     <b:LogicalName>contact</b:LogicalName>
                     <b:RelatedEntities />
                  </c:value>
               </b:KeyValuePairOfstringanyType>
            </b:Parameters>
            <b:RequestId i:nil="true" />
            <b:RequestName>Create</b:RequestName>
         </request>
      </Execute>
   </s:Body>
</s:Envelope>`

**Comment by intel352** [comment]
@shayunBBD Seems weird there's no error output from the log, just the request body. It almost seems like the request is being logged intentionally, not due to error but just for debugging purposes. Some developers use "error" log because some instances don't track info/debug logs, and rather than increase what logs are tracked, they issue an error call.

You might want to scan the open issues in the Service Cloud Connector repository to see if any match your issue, or else perhaps open a new issue yourself. I'd suggest you should perhaps create an issue report of your own as well. https://github.com/SalesforceCommerceCloud/service-cloud-connector/issues

One note though, just poking at the job that syncs customers (https://github.com/SalesforceCommerceCloud/service-cloud-connector/blob/release/b2c/sfcc/cartridges/int_service_cloud/cartridge/scripts/jobs/customerSync.js), I noticed it updates the `sscSyncResponseText` field on Profile with the results of the sync. I'd suggest you check that field to see if there's any response set whatsoever.
You can see it being set here: https://github.com/SalesforceCommerceCloud/service-cloud-connector/blob/release/b2c/sfcc/cartridges/int_service_cloud/cartridge/scripts/models/contact.js#L87  (line 87), and this function is called multiple times from the customerSync job.
